run-qa-checks.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
please rename it to `run-qa-checks` for consistency with the other modules and so that we'll be able to easily standardize it into pre-commit hooks when we're ready

Reply:
Oh, yeah sorry about that, done!